src/devices/sonoff.ts

```diff
@@ -865,6 +926,26 @@ const definitions: Definition[] = [
             await endpoint.read(0xFC11, [0x500C]);
         },
     },
+    {
+        zigbeeModel: ['ZBMicro'],
+        model: 'ZBMicro',
+        vendor: 'SONOFF',
+        description: 'Zigbee USB repeater plug',
+        ota: ota.zigbeeOTA,
+        extend: [
+            onOff(),
+            binary({
+                name: 'rf_turbo_mode',
+                cluster: 0xFC11,
+                attribute: {ID: 0x0012, type: 0x29},
+                zigbeeCommandOptions: {manufacturerCode: 0x1286},
+                description: 'Enable/disable Radio power turbo mode',
+                valueOff: ['DISABLE', 0x09],
+                valueOn: ['ENABLE', 0x14],
```

Comment:
```suggestion
                valueOff: [false, 0x09],
                valueOn: [true, 0x14],
```

Reply:
I see that there is no ZBMicro content in the newly merged version, what went wrong?

Reply:
This review comment still needs to be solved before merging, can you take a look at it?